R/api.R

```diff
@@ -84,6 +85,54 @@ endpoint_validate_survey_programme <- function() {
                               validate = TRUE)
 }
 
+returning_json_version <- function(schema = NULL, root = NULL,
+                                   status_code = 200L) {
+  ## This is the same as pkgapi::pkgapi_returning_json except we
```

Comment:
This would be the opportunity to change the schema: should this move into the payload itself? Is there a good reason for it to go into the wrapper part of the response? If *every* bit of response from hintr2 had a version field I'd say that we'd want to do something like have pkgapi allow additional fields to be added, or pass it back as a header (`X-Hintr-Version: 0.1.2`).

But if only this endpoint has a version it suggests that it belongs in the payload and we perhaps did not add it because we were trying not to break the frontend. In which case we might either add it to [ModelRunOptions](https://github.com/mrc-ide/hintr2/blob/master/inst/schema/ModelRunOptions.schema.json) or add a wrapper around ModelRunOptions with properties version/options.

We could do this in another ticket though and preserve compatibility through the migration...

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-1682

Reply:
And a ticket for the pkgapi bug we saw https://mrc-ide.myjetbrains.com/youtrack/issue/RESIDE-175